build.py

```diff
@@ -594,9 +601,6 @@ def parse_repo(base_recipe):
                             'type': 'archive',
                             **get_url_sha512('https://nodejs.org/dist/v' + nodejs_version + '/node-v' + nodejs_version + '.tar.xz')
                         }
-                    ],
```

Comment:
Why are these lines removed?

Reply:
@Lctrs I've sent you an invitation to my repo using Travis to run `build.py`, by the way thank you for your contribution for this project!